README.md

```diff
@@ -2,6 +2,18 @@
 
 Substra end-to-end (e2e) tests
 
+# A few words regarding tests design
```

Comment:
1. What about putting this section at the end of the file? I think instructions for running the tests are more important than this section (although this section is important too!)
2. What about "Test design guidelines"?

Reply:
Yes and yes!